### DISCUSSION TITLE
Hdmi - v1.3.1 - causing restart loop?

### DISCUSSION THREAD

**Discussion by chrisstonehill** [eduardofilo/RG350_adam_image#131] [Hardware problems?]
Hello. Just wondering if anyone else has experienced using hdmi from a (fantastic!) Adam image v1.3.1 equipped rg350m resulting in the handheld always restarting when shut down, after removing the hdmi connection...as seemed to happen to me?

I was happy to discover that the audio came out of the 350m still. Had got the impression there would be no audio. Simple menu seemed fine to use on my monitor. The OpenDingux loading screens looked REALLY messed up on my monitor, once a game was selected...but then all looked fine when the game was running.

After I disconnected hdmi I could not get my 350m to stay turned off. Every shut down resulted in a restart. Resolved it by holding the reset button for 30 seconds (as I saw suggested somewhere). I am reluctant to try the hdmi connection now, in case it causes the same problem.

**Comment by eduardofilo** [comment]
I've experienced those reboot loops a couple of times. In my case they occurred after a spontaneous reboot after using the sleep function several times. It was also solved by hitting reset, but I recommend that you use it just as the system finishes shutting down before the next boot, to minimize the risk of corrupting the SD file system. As seen in the video linked in this post:

https://github.com/eduardofilo/RG350_adam_image#console-cannot-be-powered-off

**Comment by chrisstonehill** [comment]
Thanks Eduardo...yes, when trying to solve the problem I saw this video of yours, having already tried the hold it down for 30 seconds idea.

So no confirmation of connecting to hdmi being a cause? Is there a recommended procedure regarding connecting and disconnecting? I've only tried it once, then steered clear due to the apparent cause of the reboot loop. Something in Simple Menu settings menu (when "start" is pushed)...yes, just looking now "PRESS A TO REBOOT AND ENABLE HDMI"...so connect the hdmi cable and THEN select this?...and then when disconnecting, turn off before removing the hdmi cable?

Just to say (and I know there are many collaborators involved...aren't there?) that the Adam Image really is a superb piece of work! Tried one other custom firmware before the Adam Image and it was full of problems. Adam Image is almost exclusively a pleasure to use. Got my 350m barely a month before the 351mp came out! Initially felt pretty stupid about that...BUT with the Adam image seems like all of my gaming needs are met...so, very many thanks to you.

**Reply by eduardofilo** [reply]
Well, the truth is that I have tried the HDMI output a few times. I do not know if it will be recommended, but it would seem reasonable to me not to connect or disconnect the HDMI cable with the system started, that is, I would connect it before turning on and disconnect it with the system already turned off. That sure causes fewer problems.
The message that appears in SimpleMenu I don't know if it is "active". I think it is just a way of saying that to activate the output you have to restart, or rather start with the cable in place.
Thanks for your words about the image, but I assure you that we have only put the pieces together. The real hard work is on ODbeta, RetroArch, SimpleMenu projects, and all the standalone emulators out there. Our job is just to install and update. Thanks...

**Reply by chrisstonehill** [reply]
Absolutely appreciate the hard work of a great many people here...but your curating and streamlining of all of these elements is an incredible service for which so many of us (who do not have the time in our stressful lives to invest) are very much indebted.

OK...I will continue to explore hdmi possibilities. My thoughts are the same as yours regarding the best, safest approach. Seems like the HDMI connection has always been a bit flakey? Didn't work at all on the first 350 devices, I think I've read. Worked OK with the stock firmware with my bought in September 2021 350m, but far from perfectly.

However, I have a Super Console X King (the Beelink box) on it's way from China, to use for big screen gaming, though having the option with my 350m is nice.
